clickhouse-jdbc/src/main/java/com/clickhouse/jdbc/ClickHouseDriver.java

```diff
@@ -69,6 +69,9 @@ public static String getFrameworksDetected() {
     }
 
     static {
+        String versionString = System.getProperty("com.clickhouse.jdbc.version", "v2");
```

Comment:
lets use another name like "com.clickhouse.jdbc.use_new_implementation" because 
- we may use com.clickhouse.jdbc.version for other reasons in the future 
- it may be too confusing